test/wdio/scoped-conditional/cmp.test.tsx

```diff
@@ -0,0 +1,64 @@
+import { h, Fragment } from '@stencil/core';
+import { render } from '@wdio/browser-runner/stencil';
+
+describe('scoped-conditional', () => {
+  beforeEach(() => {
+    function toggleHelloMessage() {
+      const scopedConditional = document.querySelector('scoped-conditional');
+      scopedConditional.renderHello = !scopedConditional.renderHello;
+    }
+
+    render({
+      template: () => (
+        <>
+          <button id="toggleHello" onclick="toggleHelloMessage()">
+            Toggle Hello Message
+          </button>
+          <scoped-conditional>
+            <div>This div will be slotted in</div>
+          </scoped-conditional>
+        </>
+      ),
+    });
+  });
+
+  it('renders the initial slotted content', async () => {
+    expect(await $('scoped-conditional div')).toHaveText(
```

Comment:
Do we need to await the assertion too?
```suggestion
    await expect(await $('scoped-conditional div')).toHaveText(
```